reciprocalspaceship/dataset.py

```diff
@@ -878,33 +878,56 @@ def compute_multiplicity(self, inplace=False, include_centering=True):
         return self
 
     @inplace
-    def assign_resolution_bins(self, bins=20, inplace=False, return_labels=True):
+    def assign_resolution_bins(
+        self,
+        bins=20,
```

Comment:
I think you should be able to supply an array of bin edges.

Reply:
New commits support this